utility/config-env.ts

```diff
@@ -92,7 +92,8 @@ export const environment = {
     allowTermsOfUse: '${process.env.ALLOW_TERMS_OF_USE}',
     minPasswordLength: '${process.env.AUTH_MIN_PASSWORD_LENGTH}',
     forceSSRServerMode: '${process.env.FORCE_SSR_SERVER_MODE}',
-    spinnerType: '${process.env.SPINNER_TYPE}',`;
+    spinnerType: '${process.env.SPINNER_TYPE}',
+    multiLang: '${process.env.MULTI_LANG}',`;
```

Comment:
Better name for this env variable could be something more explicit, like `ENABLE_MULTI_LANGUAGE`

or even `ENABLE_ANGULAR_MULTI_LANGUAGE` make avoid ambiguity at higher level on project configuration, to clarify that is only intended for angular and not for the backend

Reply:
isn't it a config variable only for the front-end UI app? If so, you won't have any disambiguity problems.

Reply:
It is, but on the container should be mapped from a variable defined on project_defaults/project_configuration/projectrc

Have a different name is possible, but the same name could be more consistent and easier. And by having the same name the above comment is then applied.

I.e. to define `multiLang: '${process.env.SOMETHING}'` in `utility/config-env.ts`, we would need

`SOMETHING: ${SOMETHING}` in `angular.yml`

We could have something consistent like:

```
multiLang: '${process.env.ENABLE_ANGULAR_MULTI_LANGUAGE}

.....

ENABLE_ANGULAR_MULTI_LANGUAGE: ${ENABLE_ANGULAR_MULTI_LANGUAGE}
```

Or eventually


```
multiLang: '${process.env.ENABLE_MULTI_LANGUAGE}

.....

ENABLE_MULTI_LANGUAGE: ${ENABLE_ANGULAR_MULTI_LANGUAGE}
```

if "angular" is redundant inside the container... but have the same name looks simpler to me


Once moved these considerations outside the container, identify the variable as "angular" only can avoid ambiguities